system/modules/comments/dca/tl_comments.php

```diff
@@ -30,8 +30,9 @@
 			'keys' => array
 			(
 				'id' => 'primary',
-				'source' => 'index',
-				'parent' => 'index'
+				'source' => 'index', // FIXME: remove?
+				'parent' => 'index', // FIXME: remove?
+				'source,parent,published' => 'index'
```

Comment:
As discussed on Mumble on March, 26th, we should remove the `source` and `parent` field indexes and add a stand-alone index on the `published` field.